SonarQube.Client.Tests/Requests/Api/V7_20/GetIssuesRequestTests.cs

```diff
@@ -0,0 +1,233 @@
+﻿/*
+ * SonarQube Client
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.Linq;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using SonarQube.Client.Api.V7_20;
+using static SonarQube.Client.Tests.Infra.MocksHelper;
+
+namespace SonarQube.Client.Tests.Requests.Api.V7_20
+{
+    [TestClass]
+    public class GetIssuesRequestTests
+    {
+        [TestMethod]
+        public async Task Invoke()
```

Comment:
Previously the GetIssues class was only tested indirectly. I've added a single direct test to cover the processing of the response and the conversion from json to the public data classes.